pcsx2-qt/MainWindow.cpp

```diff
@@ -2300,7 +2300,8 @@ void MainWindow::doSettings(const char* category /* = nullptr */)
 DebuggerWindow* MainWindow::getDebuggerWindow()
 {
 	if (!m_debugger_window)
-		m_debugger_window = new DebuggerWindow(this);
+		// Don't pass us (this) as the parent, otherwise the window is always on top of the mainwindow (on windows at least)
+		m_debugger_window = new DebuggerWindow(nullptr);
```

Comment:
If you're not setting it as the parent, you'll need to manually close/delete the debugger window when the main window is closed. Otherwise it'll both leak, as well as keep the PCSX2 process alive if the main window is closed first.

Reply:
I've amended a change to the requestExit() slot.
Does this make sense?